src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "Custom New Tab Page",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Don't bump the version as part of the PR, I'll do it separately when I do a new release.